### DISCUSSION TITLE
3.0版本当内容很多时，大纲无法导航，点击标题右边内容无法跳转到指定地方

### DISCUSSION THREAD

**Discussion by qinshiwei** [tamlok/vnote#1770] [General]
3.0版本当内容很多时，大纲无法导航，点击标题右边内容无法跳转到指定地方。请问是咋回事？

**Comment by tamlok** [comment]
给一个示例文件？

**Reply by qinshiwei** [reply]
感谢闪电回复！测试了下，当目录下面有vswp缓存文件时，好像就无法正常使用大纲，删除后，目前又恢复了，如果后续再出现，我把文档共享给你。感谢！产品非常棒！长期以来我都在寻找一个文本文件可以做大纲的软件，终于在此得偿所愿，非常感谢！

**Reply by ZhiboWong** [reply]
> 给一个示例文件？

https://gitee.com/Gilbert_Wong/it_blog/blob/master/NetworkDevices/%E5%8D%8E%E4%B8%BA/HCIA.md
比如说这个文件，内容很多，大纲跳转就没反应了。
